src/frontend/posts_page/components/ArticlePage.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+class ArticlePage extends React.Component {
+
+    render() {
+        return (
+            <div>
```

Comment:
Is this used anywhere?

Reply:
This well be used later. It is here because it is needed in the future, but just not implemented yet.